hazelcast/src/main/java/com/hazelcast/query/impl/getters/GetterFactory.java

```diff
@@ -47,6 +49,9 @@ public static Getter newFieldGetter(Object object, Getter parentGetter, Field fi
                 returnType = getCollectionType(collection);
             }
             if (returnType == null) {
+                if (modifierSuffix.equals(ANY_POSTFIX)) {
+                    return NullMultiValueGetter.NULL_MULTIVALUE_GETTER;
```

Comment:
`NullMultiValueGetter.INSTANCE`?
 or can be imported statically, current form can make reads hard